hail/python/hail/docs/change_log.md

```diff
@@ -318,7 +318,7 @@ Released 2023-05-30
 
 ## Version 0.2.117
 
-Released 2023-05-19
+Released 2023-05-23
```

Comment:
I think this is `2023-05-22` -- could be a timezone difference.

Reply:
It was a toss-up between the datestamp of the commit (Mon May 22 21:34:45 2023 -0400) and the datestamp of the tag (Tue May 23 02:19:17 2023 +0000).

However on revisiting this I notice the +0000: the automation-based tagging does not get set to appear as being done in Hail's local timezone and actually occurred within an hour of the commit, indeed on your 22nd. Thanks.